### ISSUE TITLE
TypeError if calling from Chrome (Audio only) to Edge with Version 4.2.1

### ISSUE THREAD

**Issue by notz** [webrtcHacks/adapter#638]
**Browsers and versions affected**
Chrome (60)
Edge (15063)

**Description**
If calling from Chrome to Edge with audio only createAnswer fails with an TypeError: "Unable to get property '0' of undefined or null reference in Line 2989

Which is this block:
````
// FIXME: look at direction.
      if (transceiver.stream) {
        var localTrack;
        if (transceiver.kind === 'audio') {
          localTrack = transceiver.stream.getAudioTracks()[0];
        } else if (transceiver.kind === 'video') {
          localTrack = transceiver.stream.getVideoTracks()[0];
        }
        if (localTrack) {
          // add RTX
          if (edgeVersion >= 15019 && transceiver.kind === 'video') {
            transceiver.sendEncodingParameters[0].rtx = {
              ssrc: (2 * sdpMLineIndex + 2) * 1001 + 1
            };
          }
        }
      }
````
Line 2989 -> transceiver.sendEncodingParameters[0]

My sdp from chrome:
````
v=0
o=- 6115746443906320221 2 IN IP4 127.0.0.1
s=-
t=0 0
a=group:BUNDLE audio
a=msid-semantic: WMS 931d345f-1032-4003-bd2f-ccba9ea57d7a
m=audio 9 UDP/TLS/RTP/SAVPF 111 103 104 9 0 8 106 105 13 110 112 113 126
c=IN IP4 0.0.0.0
a=rtcp:9 IN IP4 0.0.0.0
a=ice-ufrag:CJ/N
a=ice-pwd:+lyXO56u3ZaSG3bxq7VgdkkD
a=fingerprint:sha-256 FF:7E:06:DC:60:7F:69:C6:38:13:B7:78:D5:27:B6:46:6E:34:68:9A:50:50:F5:F9:49:49:EF:F5:8E:B4:0A:39
a=setup:actpass
a=mid:audio
a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
a=sendrecv
a=rtcp-mux
a=rtpmap:111 opus/48000/2
a=rtcp-fb:111 transport-cc
a=fmtp:111 minptime=10;useinbandfec=1
a=rtpmap:103 ISAC/16000
a=rtpmap:104 ISAC/32000
a=rtpmap:9 G722/8000
a=rtpmap:0 PCMU/8000
a=rtpmap:8 PCMA/8000
a=rtpmap:106 CN/32000
a=rtpmap:105 CN/16000
a=rtpmap:13 CN/8000
a=rtpmap:110 telephone-event/48000
a=rtpmap:112 telephone-event/32000
a=rtpmap:113 telephone-event/16000
a=rtpmap:126 telephone-event/8000
a=ssrc:588539270 cname:TKNwqHtSDon7Uh6O
a=ssrc:588539270 msid:931d345f-1032-4003-bd2f-ccba9ea57d7a 662d7974-cd61-4dbf-964a-dde626ca7a0c
a=ssrc:588539270 mslabel:931d345f-1032-4003-bd2f-ccba9ea57d7a
a=ssrc:588539270 label:662d7974-cd61-4dbf-964a-dde626ca7a0c
````

**Steps to reproduce**
Calling from Chrome (audio only) to Edge (audio + video)

**Expected results**
Working call

**Actual results**
Exception in createAnswer

**Comment by fippo** [comment]
Can't reproduce with https://jsfiddle.net/dfn8yqbp/1/ -- does that not work for you?

**Comment by notz** [comment]
Yes, but this not: https://jsfiddle.net/dfn8yqbp/2/ (only enabled video). Only Chrome has no video but Edge has Video.

**Comment by fippo** [comment]
ah... that makes sense. Will try to get it fixed in time for the next release!

**Comment by fippo** [comment]
Thanks, the bug that caused this was quite fundamental!
https://www.npmjs.com/package/rtcpeerconnection-shim@1.0.2 fixes the issue -- that will be used after the next major release.

@KaptenJansson can we do the major release soon? ;-)

**Comment by notz** [comment]
@fippo This means it's not included in 4.2.2 and will be included in 4.3.0?

**Comment by fippo** [comment]
it will be in 5.0.0
